cellpack/autopack/ingredient/grow.py

```diff
@@ -7,7 +7,6 @@
 from panda3d.core import Point3, TransformState
 from panda3d.bullet import BulletSphereShape, BulletRigidBodyNode
 from random import uniform, gauss, random
-from time import time
```

Comment:
Is this related to rapid or just an unnecessary import?

Reply:
not related to rapid, used in development for timing the execution of the growing algorithm.